src/components/Form.tsx

```diff
@@ -63,13 +78,62 @@ export const Form: React.FunctionComponent<IFormtronControl> = ({
               }}
               fieldComponents={fieldComponents}
               disabled={disabled}
+              layout={layout}
             />
-          </div>
+          </Box>
         );
-        return el;
-      })}
-    </div>
-  );
+      }
+    }
+    if (cells.length > 0) {
+      innerStuff.push(<Flex>{cells}</Flex>);
+    }
+  }
+  // const innerStuff = (
+  //   <div
```

Comment:
Remove this commented out section

Reply:
oops
didn't see your comment in time 😊

Reply:
yah that's embarrassing whoops. Oh vell.